src/main/java/org/spongepowered/api/text/selector/SelectorTypes.java

```diff
@@ -24,56 +24,33 @@
  */
 package org.spongepowered.api.text.selector;
 
-import com.google.common.base.Optional;
-
-import java.util.List;
-
 /**
- * SelectorTypes is a list of the default selector types that are
- * available in Vanilla Minecraft.
+ * SelectorTypes is a list of the default selector types that are available in
+ * Vanilla Minecraft.
  */
 public final class SelectorTypes {
 
+    private SelectorTypes() {
+    }
+
     /**
      * The all players selector type.
      */
     public static final SelectorType ALL_PLAYERS = null;
+
     /**
      * The all entities selector type.
      */
     public static final SelectorType ALL_ENTITIES = null;
+
     /**
      * The nearest player selector type.
      */
     public static final SelectorType NEAREST_PLAYER = null;
+
     /**
      * The random player selector type.
      */
     public static final SelectorType RANDOM_PLAYER = null;
-    private static final SelectorTypeFactory factory = null;
-
-    private SelectorTypes() {
-    }
-
-    /**
-     * Gets the {@link SelectorType} with the specified ID.
```

Comment:
This message was created automatically by mail delivery software.

A message that you sent could not be delivered to one or more of its
recipients. This is a temporary error. The following address(es) deferred:

  adam@eq2.serveftp.net
    Domain eq2.co.uk has exceeded the max emails per hour (63/60 (105%)) allowed.  Message will be reattempted later

------- This is a copy of the message, including all the headers. ------
Received: from o3.sgmail.github.com ([192.254.112.98]:35597)
	by web3.cov.yesit.co.uk with esmtps (TLSv1.2:DHE-RSA-AES128-GCM-SHA256:128)
	(Exim 4.84)
	(envelope-from <bounces+848413-d0b0-adam=eq2.co.uk@sgmail.github.com>)
	id 1YNeZE-0003Wm-Ih
	for adam@eq2.co.uk; Tue, 17 Feb 2015 09:35:44 +0000
DKIM-Signature: v=1; a=rsa-sha1; c=relaxed; d=github.com; 
	h=from:reply-to:to:cc:in-reply-to:references:subject:mime-version:content-type:content-transfer-encoding:list-id:list-archive:list-post:list-unsubscribe; 
	s=s20150108; bh=6FD9a2VCB8hJhB5IsQIN7tusHAU=; b=hcJtmI+3MXnp4Qaq
	8Yl48kA1S7qy8DT+fBbIrOGMguWBKJIlwTHHt13/tUlapA14azJVbg26Yg3oG/yO
	1QrjhHrdKvz5vIBC6Og0yn8UuYG4rifDuSAC1P+eBbVaiPGhSHDaA3iSQGq3mt2G
	v+5OIuYLYv9MOQrCKWnS9SaYAfU=
Received: by filter0207p1mdw1.sendgrid.net with SMTP id filter0207p1mdw1.13135.54E30B6E4
        2015-02-17 09:35:42.99384668 +0000 UTC
Received: from github-smtp2b-ext-cp1-prd.iad.github.net (github-smtp2b-ext-cp1-prd.iad.github.net [192.30.253.17])
	by ismtpd-022 (SG) with ESMTP id 14b96e4a95b.2d90.213b78
	for <adam@eq2.co.uk>; Tue, 17 Feb 2015 09:35:42 +0000 (UTC)
Date: Tue, 17 Feb 2015 01:35:42 -0800
From: gratimax <notifications@github.com>
Reply-To: SpongePowered/SpongeAPI <reply@reply.github.com>
To: SpongePowered/SpongeAPI <SpongeAPI@noreply.github.com>
Cc: Mumfrey <adam@eq2.co.uk>
Message-ID: <SpongePowered/SpongeAPI/pull/425/r24803050@github.com>
In-Reply-To: <SpongePowered/SpongeAPI/pull/425@github.com>
References: <SpongePowered/SpongeAPI/pull/425@github.com>
Subject: Re: [SpongeAPI] [WIP] Text API 2 (#425)
Mime-Version: 1.0
Content-Type: multipart/alternative;
 boundary="--==_mimepart_54e30b6ec79f4_15673fa2c016d29c7487d0";
 charset=UTF-8
Content-Transfer-Encoding: 7bit
Precedence: list
X-GitHub-Sender: gratimax
X-GitHub-Recipient: Mumfrey
X-GitHub-Reason: comment
List-ID: SpongePowered/SpongeAPI <SpongeAPI.SpongePowered.github.com>
List-Archive: https://github.com/SpongePowered/SpongeAPI
List-Post: <mailto:reply@reply.github.com>
List-Unsubscribe: <mailto:unsub+001180bf45b1d78f7c28d574c7286edb21f34f4fde14bcf892cf0000000110facd6e92a170ce017a76ea@reply.github.com>,
 <https://github.com/notifications/unsubscribe/ABGAv4nGwDXoyKn9q4JQxLAlhtEwCSA2ks5nswLugaJpZM4DZr_C>
X-Auto-Response-Suppress: All
X-GitHub-Recipient-Address: adam@eq2.co.uk
X-SG-EID: VzkE0stiinYMPYWPWpnZ4OvH/Ru8MrIuYzb7JqhnFX8zcYSSKMymj84W6gudzXYijJbvB+/046dLkh
 OrRBoeW7gv7x31p0FwdgHCF7DlTp/dwm8Xv3DbYGBtFpZV6HBuQgwNi16r2mFalHzAmB78d35X9Rti
 Ot1cNtn3xSv1Hv4=

----==_mimepart_54e30b6ec79f4_15673fa2c016d29c7487d0
Content-Type: text/plain;
 charset=UTF-8
Content-Transfer-Encoding: 7bit

>      /**
>       * The random player selector type.
>       */
>      public static final SelectorType RANDOM_PLAYER = null;
> -    private static final SelectorTypeFactory factory = null;
> -
> -    private SelectorTypes() {
> -    }
> -
> -    /**
> -     * Gets the {@link SelectorType} with the specified ID.

why were these removed?

---
Reply to this email directly or view it on GitHub:
https://github.com/SpongePowered/SpongeAPI/pull/425/files#r24803050
----==_mimepart_54e30b6ec79f4_15673fa2c016d29c7487d0
Content-Type: text/html;
 charset=UTF-8
Content-Transfer-Encoding: 7bit

<p>In <a href="https://github.com/SpongePowered/SpongeAPI/pull/425#discussion_r24803050">src/main/java/org/spongepowered/api/text/selector/SelectorTypes.java</a>:</p>
<pre style='color:#555'>&gt;      /**
&gt;       * The random player selector type.
&gt;       */
&gt;      public static final SelectorType RANDOM_PLAYER = null;
&gt; -    private static final SelectorTypeFactory factory = null;
&gt; -
&gt; -    private SelectorTypes() {
&gt; -    }
&gt; -
&gt; -    /**
&gt; -     * Gets the {@link SelectorType} with the specified ID.
</pre>
<p>why were these removed?</p>

<p style="font-size:small;-webkit-text-size-adjust:none;color:#666;">&mdash;<br>Reply to this email directly or <a href="https://github.com/SpongePowered/SpongeAPI/pull/425/files#r24803050">view it on GitHub</a>.<img alt="" height="1" src="https://github.com/notifications/beacon/ABGAv4Pz6tk_MoqQBTPmJXcM8VHvy_mIks5nswLugaJpZM4DZr_C.gif" width="1" /></p>
<div itemscope itemtype="http://schema.org/EmailMessage">
  <div itemprop="action" itemscope itemtype="http://schema.org/ViewAction">
    <link itemprop="url" href="https://github.com/SpongePowered/SpongeAPI/pull/425/files#r24803050"></link>
    <meta itemprop="name" content="View Pull Request"></meta>
  </div>
  <meta itemprop="description" content="View this Pull Request on GitHub"></meta>
</div>

----==_mimepart_54e30b6ec79f4_15673fa2c016d29c7487d0--